tidb/datadog_checks/tidb/check.py

```diff
@@ -0,0 +1,74 @@
+from datadog_checks.base import OpenMetricsBaseCheck
+
+from .metrics import GO_RUNTIME_METRICS, PD_METRICS, TIDB_METRICS, TIKV_METRICS
+
+
+class TiDBCheck(OpenMetricsBaseCheck):
+    DEFAULT_METRIC_LIMIT = 0
```

Comment:
```suggestion
    __NAMESPACE__ = 'tidb'
```
Let's not override the `DEFAULT_METRIC_LIMIT`
Also, `__NAMESPACE__` should add a `tidb.` prefix to all the metrics submitted by the check